README.md

```diff
@@ -5,6 +5,56 @@
 
 An Android app for recording blood pressure measurements.
 
+## Pre-requisites
+
+The application currently requires JDK 1.8 to build. If you already have JDK 1.8 installed, skip this step.
+
+**I am not sure if I have the JDK installed, or I am not sure if I have the right version installed**
+
+Run the command `java -version`. If you have the right version of the JDK installed, you should see something like:
+```sh
+openjdk version "1.8.0_232"
+OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_232-b09)
+OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.232-b09, mixed mode)
+```
+
+If this command has an error, or shows a different version, you can follow the instructions below to install the JDK.
```

Comment:
I think the answers to the questions do not need a margin on the left.
Wdyt?

Reply:
I have not added any margins, I think this is how it's getting rendered. Could you please check the raw file and confirm?

Reply:
Ok. No worries then.